eesp-ikev2.org

```diff
@@ -66,6 +66,12 @@ in stateful decryption configurations, sharing a common SPI namespace
 while introducing new capabilities to enhance IPsec’s performance
 and versatility in modern use cases.
 
+EESP Sub SAs provide a unidirectional SA instead of conventional
```

Comment:
THis part could be argued by others. Conventional Child SAs are unidirectional too, they are just always creted in pairs by IKEv2. But this is not EESP limitation - this is IKEv2 limitation. More accurate wording is needed.